src/bots/modchat/constants.js

```diff
@@ -0,0 +1,18 @@
+import { dedent } from '../../utils';
+
+/**
+ * Contructs string object with options
+ * @param  {...any} options - data for strings and other options
+ * @return {Object.<string, string>} - string object with all AMA bot strings
+ */
+export const getStrings = (...options) => {
```

Comment:
not sure i understand why you're spreading them here. you could just keep them as an array since either way you'll have to grab them with an index.